docs/examples/use_cases/webdataset-externalsource.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<p>Looks like initial fill is 256 by default.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>